koji_wrapper/tag.py

```diff
@@ -78,6 +78,6 @@ def _filter_tagged(self, tagged_builds):
 
         return self.tagged_list
 
-    def latest_by_tag(self):
+    def latest_by_nvr(self):
```

Comment:
Is there a unittest for calling this method that should also be updated?

Reply:
this method is not yet implemented, so, no, no test yet

Reply:
if not implemented yet, do we want to include the interface yet then? or just leave it out till it is implemented along with the unit tests for it?

Reply:
The interface is not expected to be stable until a 1.0 release (though I expect this should not have to change), so I don't see any problem with a placeholder, personally.